### DISCUSSION TITLE
Add the directive ProxyPassReverse tot Apache reverse proxy?

### DISCUSSION THREAD

**Discussion by Verhoeckx** [nextcloud/all-in-one#2023] [Ideas] [answered]
Hi Nextcloud AIO developers!

I'm not an Apache expert, but according the official documentation of Apache the directive [ProxyPassReverse](https://httpd.apache.org/docs/2.2/mod/mod_proxy.html#ProxyPassReverse) should be added when Apache is being used as a reverse proxy. So shouldn't this be added to [the example of the Apache reverse proxy](https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#apache)?

Just my two cent ;-),

Verhoeckx

**Comment by Zoey2936** [comment]
I think it could be added, but it works without it very good in my tests

**Reply by Verhoeckx** [reply]
That's true, it also works without it, but according the documentation it should be added: 

> This is essential when Apache is used as a reverse proxy (or gateway) to avoid bypassing the reverse proxy because of HTTP redirects on the backend servers which stay behind the reverse proxy.

The documentation ask for improvements (Disclaimer: It might be possible that the config below is not working 100% correctly, yet. Improvements to it are very welcome!), so I thought I could mention it.

**Reply by Zoey2936** [reply]
does it work, when this option is set?

**Reply by Zoey2936** [reply]
if yes, simply create an PR, please

**Reply by Verhoeckx** [reply]
> does it work, when this option is set?

Yes, everything seems to work normally!

**Reply by Verhoeckx** [reply]
> if yes, simply create an PR, please

How/where do I that?

**Reply by Zoey2936** [reply]
Here: https://github.com/nextcloud/all-in-one/edit/main/reverse-proxy.md

**Reply by Verhoeckx** [reply]
Thanks!!

**Reply by Verhoeckx** [reply]
> if yes, simply create an PR, please

I have done so!

**Comment by szaimen** [comment, marked as answer]
Resolved with https://github.com/nextcloud/all-in-one/pull/2024
